app/lib/tool/neat_task/pub_dev_tasks.dart

```diff
@@ -52,7 +54,21 @@ void setupAnalyzerPeriodicTasks() {
 /// Setup the tasks that we are running in the dartdoc service.
 void setupDartdocPeriodicTasks() {
   _setupJobCleanupPeriodicTasks();
-  // TODO: migrate cleanup of the extracted SDK data
+
+  // Deletes the extracted dartdoc data from old SDKs.
+  _daily(
```

Comment:
If this involves crawling around in GCS then maybe we don't need to run it daily?

Reply:
At this point this is a cheap operation: scanning a list of objects in a directory with less than 100 items, and this way it will be done fewer times than right now (every worker startup would trigger it).

Let's review it again once we have all the periodic task migrated. I've added another bullet point to the tracking issue.